.github/workflows/fprime-gds-tests.yml

```diff
@@ -0,0 +1,23 @@
+name: fprime-gds tests
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          pip install -e .
+          python -m pip install --upgrade pip
+          pip install pytest
```

Comment:
Is there a reason that you're manually installing pytest? It's a package dependency so it should automatically be installed with the rest of the GDS